### DISCUSSION TITLE
Rendering distance

### DISCUSSION THREAD

**Discussion by konrad-thoc** [sceneview/sceneview#548] [General]
Currently when I insert a 3D node at a distance of more than 30m it is not visible, can I somehow increase the drawing area? In the SceneForm it looked like this, but I cannot find an alternative in SceneView.

arFragment.getArSceneView().getScene().getCamera().setFarClipPlane(500)

**Comment by konrad-thoc** [comment]
I found the right one we can use `sceneView.cameraNode.far = 100f`

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, the far clipping plane can be configured through Filament's camera settings. By default, SceneView sets reasonable near/far clip distances, but you can adjust them for larger scenes:

The camera's near/far planes determine what range of distances are rendered. For large outdoor scenes or very close-up views, you may need to adjust these. Access the Filament camera through the scene to configure clip distances.

Note: In AR mode (`ARSceneView`), the camera projection is driven by ARCore to match the physical camera, so clip distances are managed automatically.
